### PR TITLE
[MODULAR?] Starts updating the jobs to match currently known factions

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request
Will be elaborated once I know all the companies.
Opening now, so people can see what i'm doing, and xyel can yell at me if a company doesn't exist

## How This Contributes To The Skyrat Roleplay Experience
Ditto

## Changelog

:cl:
not yet
/:cl:

### CODE REVIEW

**Comment by theselfish** [comment]
This just clutters up the names.

I think the new alt titles are fine, but adding FTU to the start just...Throws the balance off. No other job abbreviates things.

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 1]
```diff
@@ -40,7 +40,7 @@
 	alt_titles = list("Captain", "Station Commander", "Commanding Officer", "Site Manager")
 
 /datum/job/cargo_technician
-	alt_titles = list("Cargo Technician", "Deck Worker", "Mailman")
+	alt_titles = list("FTU Supply Technician", "FTU Deck Worker", "FTU Mailman", "FTU Merchant", "FTU Supply Operative")
```
Comment (Draganfrukts) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
	alt_titles = list("Union Supply Technician", "Union Deck Worker", "Union Mailman", "Union Merchant", "UnionSupply Operative")
```

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 2]
```diff
@@ -115,7 +115,7 @@
 	alt_titles = list("Psychologist", "Psychiatrist", "Therapist", "Counsellor")
 
 /datum/job/quartermaster
-	alt_titles = list("Quartermaster", "Deck Chief", "Supply Foreman", "Head of Supply", "Chief Requisition Officer", "Logistics Officer")
+	alt_titles = list("FTU Quartermaster", "FTU Deck Chief", "FTU Supply Foreman", "FTU Head of Supply", "FTU Chief Requisition Officer", "FTU Logistics Officer")
```
Comment (Draganfrukts) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
	alt_titles = list("Union Quartermaster", "Union Deck Chief", "Union Supply Foreman", "Union Head of Supply", "Union Chief Requisitions Officer", "Union Logistics Officer")
```

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 3]
```diff
@@ -147,7 +147,7 @@
 	alt_titles = list("Security Officer", "Security Operative", "Peacekeeper")
 
 /datum/job/shaft_miner
-	alt_titles = list("Shaft Miner", "Excavator", "Spelunker", "Drill Technician", "Prospector")
+	alt_titles = list("FTU Shaft Miner", "FTU Excavator", "FTU Spelunker", "FTU Drill Technician", "FTU Prospector")
```
Comment (Draganfrukts) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
	alt_titles = list("Union Miner", "Union Excavator", "Union Spelunker", "Union Drill Technician", "Union Prospector")
```

**Review by Draganfrukts — CHANGES_REQUESTED** [review]
I think it's better if we say "Union" instead of abbreviating Free Trade Union in every instance

**Comment by RatFromTheJungle** [comment]
> This just clutters up the names.
> 
> I think the new alt titles are fine, but adding FTU to the start just...Throws the balance off. No other job abbreviates things.

they'll all have signifying starts, which, long-term will help with actually finding out who's in a department, (for ex: try to find a roboticist in the messenger when they give themselves a name like 'Biomechanical Engineer', as compared to like '[science signifier] Biomechanical Engineer' ) which would, also, convienently, sort them by department in the message list

**Comment by theselfish** [comment]
> > This just clutters up the names.
> > I think the new alt titles are fine, but adding FTU to the start just...Throws the balance off. No other job abbreviates things.
> 
> they'll all have signifying starts, which, long-term will help with actually finding out who's in a department, (for ex: try to find a roboticist when they give themselves a name like 'Biomechanical Engineer', as compared to like '[science signifier] Biomechanical Engineer' ) which would, also, convienently, sort them by department in the message list

Adding Union, FTO or Whatever to the start of all the names is only going to beg the question "Where's the normal one?"

If you added Funny Clown to the start of Clown, people would be wondering where the normal clown is.

**Comment by RatFromTheJungle** [comment]
> > > This just clutters up the names.
> > > I think the new alt titles are fine, but adding FTU to the start just...Throws the balance off. No other job abbreviates things.
> > 
> > 
> > they'll all have signifying starts, which, long-term will help with actually finding out who's in a department, (for ex: try to find a roboticist when they give themselves a name like 'Biomechanical Engineer', as compared to like '[science signifier] Biomechanical Engineer' ) which would, also, convienently, sort them by department in the message list
> 
> Adding Union, FTO or Whatever to the start of all the names is only going to beg the question "Where's the normal one?"
> 
> If you added Funny Clown to the start of Clown, people would be wondering where the normal clown is.

they're uh.. gone, that's the jist of it

the blank titles (default) will always carry that 'works for NT' stigma, that still exists even in departments whom were supposed to be seperated from NT (like cargo has been for like a year), I wouldn't mind like, free-lancer variants, or maybe finding a better way to addendum the titles to ids (maybe a second thing in parenthesis? like John Smith (Cargo Technician) (FTU) ) but, currently, this's the system we got
and, if you read the desc, it won't stop at cargo, it's just that some jobs have names so rediculously long, and the company they belong to isn't in the public lore yet
image

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 4]
```diff
@@ -147,7 +147,7 @@
 	alt_titles = list("Security Officer", "Security Operative", "Peacekeeper")
 
 /datum/job/shaft_miner
-	alt_titles = list("Shaft Miner", "Excavator", "Spelunker", "Drill Technician", "Prospector")
+	alt_titles = list("Union Miner", "Union Excavator", "Union Spelunker", "Union Drill Technician", "Union Prospector")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: Do they *all* need to have Union in the name, though? I feel like it'll mostly just make all the alt-titles longer and more complicated for very little gain.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm (hunk 4 above)
Comment (RatFromTheJungle) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: > Do they _all_ need to have Union in the name, though? I feel like it'll mostly just make all the alt-titles longer and more complicated for very little gain.

i'm talking to someone about a change for the better, that being, it'll addendum it after the title, like
John Jackson, (Shaft Miner) (FTU) (and also cause 'sort by job' to organize them by department in the pda messenger)

which is why it's still in draft

**Comment by RatFromTheJungle** [comment]
closing so stale bot goes away go away stalebot

**Pull request closed without merging**